python/src/iceberg/expressions/literals.py

```diff
@@ -44,13 +44,7 @@
     TimeType,
     UUIDType,
 )
-from iceberg.utils.datetime import (
```

Comment:
I feel like the format before looks more clear to me, what do you think?

Reply:
I somewhat agree in this case, but I'm of the opinion that if this is how `black` / the formatting tool formats, we should choose to opt for the chosen default styles unless the change is really glaring.

I imagine there's a limit we can maybe configure but 4 is arguably enough to use this format. At 6 or so I would definitely prefer this style.

My understanding is that this is an area where convention (as dictated by the tool) is usually preferred over configuration.

Reply:
On further thought, if this is easily configurable I'd be open to it.

Looking at all of the import changes, I do seem to prefer the old style more. But possibly we'll adjust?

Reply:
Hey @dramaticlly @kbendick thanks for the input, and I agree that breaking it up into multiple lines is more readable.

This changed because I also set isort to a max line length of 130 chars. Now I've set it to when there are more than 4 imports, it will break it up into a multi-line statement.

This is configurable by isort, and I've just updated the code accodingly. Black is totally fine with it, so that should be okay 👍🏻